app/grants/models/subscription.py

```diff
@@ -186,6 +186,14 @@ class Subscription(SuperModel):
     def negative(self):
         return self.is_postive_vote == False
 
+    @property
+    def match_amount(self):
+        return 4.4 # TODO: actually calc it
+
+    @property
+    def match_amount_token(self):
+        return 'DAI'
```

Comment:
will match amounts always be DAI?  would be nice to future proof this in case we ever do matching in diff tokens

Reply:
it would always be a stable coin! for the forseable future so i'd say this is ok